jetty-ee11/jetty-ee11-servlet/src/test/java/org/eclipse/jetty/ee11/servlet/ResponseHeadersTest.java

```diff
@@ -734,6 +734,58 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
         assertThat(response.getContent(), containsString("Hello"));
     }
 
+    @Test
+    public void testAssumedContentType() throws Exception
```

Comment:
Need a test that calls `setLocale()` _after_ one of these assumed content types that cannot have a character encoding to verify that we do the right thing (whatever that is!).